config/swagger.yaml

```diff
@@ -1461,7 +1461,7 @@ definitions:
     - data_schema
   SingleTxObject:
     allOf:
-      - $ref: '#/definitions/TxHashOrObject'
+      - $ref: '#/definitions/SingleTxHashOrObject'
```

Comment:
I see this main change.

Reply:
Yes, this and a couple of modifications in python code